modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -7,16 +7,14 @@
 = vSphere automatic migration
 
 == New installations of {product-title}
-For new installations of {product-title} 4.13, or later, automatic migration is enabled by default.
+For new installations of {product-title} 4.14, or later, automatic migration is enabled by default.
 
 == Upgrading {product-title}
-When upgrading from {product-title} 4.12, or earlier, to 4.13, automatic CSI migration for vSphere only occurs if you opt in.
+If you have vSphere in-tree persistent volumes (PVs) and want to upgrade from {product-title} 4.12 or 4.13 to 4.14, then you must upgrade to vSphere 7.0 Update 3L or 8.0 Update 2, or later. After upgrading vSphere, your {product-title} upgrade can occur and automatic migration is enabled by default.
 
 [IMPORTANT]
 ====
-Carefully review the following consequences before opting in to migration:
-
-* link:https://access.redhat.com/node/7011683[There are known issues that can occur that can cause the migration to fail].
+Migration includes significant consequences:
```

Comment:
This note looks pretty scary. Where did we get wording for: 

```
Migration is a disruptive process. The Machine Config Operator (MCO) has to update kubelet, which means degrading each node as it rolls out the new MachineConfig to each machine.
```

?

Reply:
Docs have been published: 
4.13: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/storage/using-container-storage-interface-csi#persistent-storage-csi-google-cloud-file
4.12: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/storage/using-container-storage-interface-csi#persistent-storage-csi-google-cloud-file

Reply:
@gnufied @gcharot Not sure who provided this wording. Gregory or Eric maybe? 

Do you have a recommendation on how you would like it reworded? I would have to also go back and change it in the 4.13 docs, if so.

Reply:
I don't remember adding this.
Regardless, since we don't have it for other drivers, I think we should remove it unless @gnufied thinks we need to highlight something specific that does not apply to vsphere.

Reply:
Yeah we should remove it altogether in 4.14

Reply:
@gnufied Should I remove this same bullet point from 4.13? I'm updating that section of the docs anyway on a related PR to cover the upgrade 4.13->4.14 issues.

Reply:
Yes it can be removed from 4.13. But not from 4.12 I guess where migration is still TP for vSphere.